bin/curi_builder/builder/__init__.py

```diff
@@ -137,6 +138,8 @@ def set_status(context, status, sha, token):
 
 
 def post_pr_comment(pr_number, comment, token):
```

Comment:
and just a reminder that builder images will need to be manually pushed to modl and prod, this one doesn't get built in any of the workflows